neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/SampleIT.java

```diff
@@ -33,6 +33,7 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.logging.LogManager;
```

Comment:
nit: can we clean it?

Reply:
Yes, I was trying to mute logs, but decided to do that another PR
Forgot to clean up the rest